Cmfcmf/OpenWeatherMap.php

```diff
@@ -94,7 +102,7 @@ class OpenWeatherMap
      * @throws \Exception If $cache is neither false nor a valid callable extending Cmfcmf\OpenWeatherMap\Util\Cache.
      * @api
      */
-    public function __construct($fetcher = null, $cacheClass = false, $seconds = 600)
+    public function __construct($appid = null, $fetcher = null, $cacheClass = false, $seconds = 600)
```

Comment:
Make sense. I think it wouldn't make sense to put it at the end either because then you can't assume the rest of the defaults. This leaves us only with the option of explicitly setting it with setApiKey() after the constructor is called.